general/overlay/etc/init.d/S70vendor

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# load vendor specific drivers
+# 
+vendor=$(ipcinfo -v)
+
+case "$1" in
+  start)
+	logger -s -p daemon.info -t ${vendor} "Check MAC for ${vendor} devices..."
+	logger -s -p daemon.info -t ${vendor} "$(check_mac)"
```

Comment:
Logging could be refactored further to a function. Something like
```
log_vendor() {
  logger -s -p daemon.info -t ${vendor} "$1"
}
```

Reply:
I would also replace messages.
`Check MAC ... `- that line is not necessary at all
`The eth0 interface has a correct MAC` - remove
Idea is to leave only 'error' message, and replace that message as follows:
`The eth0 interface has a lousy MAC, please change it..`  -> `Warning. Wired network interface has default MAC address, please change it.`